deployments/ansible/roles/signalfx-agent/tasks/debian_repo.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Add an Apt signing key for Signalfx Agent
+  apt_key:
+    url: "{{ debian_gpg_key_url }}"
+    keyring: /etc/apt/trusted.gpg.d/signalfx.gpg
+    state: present
+
+- name: Add Signalfx Agent repository into sources list
+  apt_repository:
+    repo: "{{ debian_repo_url }}"
```

Comment:
Construct this string directly here instead of using a separate variable.  There is really no need to since the value is only used once.

Reply:
yeah, updated